mantidproject/mantid
# Indirect - Elastic Window Multiple - Normalizing by lowest temperature

Temperatures from the ElasticWindowMultiple algorithm are currently not being normalised by lowest temperature.

Changed retrieving the first temperature in the list of temperatures, to finding the minimum.

**To test:**
1. Open Mantid release 3.10.
2. Navigate to Interfaces > Indirect > Data Analysis and open the Elwin tab.
3. In the 'Input File' field, provide *_red.nxs files which were reduced with a temperature log (files provided below).
4. Ensure 'Normalize to Lowest Temp' is checked.
5. Run the algorithm.
6. Save the resultant workspace.
7. Open Mantid from this branch.
8. Repeat steps 2-5, ensuring the same workspace is used.
9. Load the previously saved file into the ADS.
10. View both workspace - if the lowest temperature was not the first y-value in the original *_red.nxs file, these workspaces should be different - else they should be the same.

Test files:
[Elwin Files.zip](https://github.com/mantidproject/mantid/files/1176302/Elwin.Files.zip)



Fixes #20039 

**Release Notes** 

Added fix to the release notes.
---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
